docs/modules/ROOT/pages/servlet/authentication/passwords/digest.adoc

```diff
@@ -55,15 +55,15 @@ DigestAuthenticationEntryPoint entryPoint() {
 DigestAuthenticationFilter digestAuthenticationFilter() {
 	DigestAuthenticationFilter result = new DigestAuthenticationFilter();
 	result.setUserDetailsService(userDetailsService);
-	result.setAuthenticationEntryPoint(entryPoint());
+	result.setAuthenticationEntryPoint(authenticationEntryPoint());
 }
 
 @Bean
 public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 	http
 		// ...
 		.exceptionHandling(e -> e.authenticationEntryPoint(authenticationEntryPoint()))
-		.addFilterBefore(digestFilter());
+		.addFilterAfter(digestAuthenticationFilter(), ExceptionTranslationFilter.class);
```

Comment:
This method should be `.addFilter(digestAuthenticationFilter())` since the `DigestAuthenticationFilter` has an entry in `FilterOrderRegistration`.